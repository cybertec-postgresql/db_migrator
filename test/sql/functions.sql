\i test/psql.sql

SELECT plan(3);

SELECT is(
    db_migrate_functions(plugin => 'noop_migrator'),
    0,
    'Should create functions without errors'
);

SELECT has_function('sch1', t.function_name, t.args)
FROM (
    VALUES ('multiply', ARRAY['int', 'int']),
           ('divide', ARRAY['int', 'int'])
) AS t(function_name, args);

SELECT * FROM finish();