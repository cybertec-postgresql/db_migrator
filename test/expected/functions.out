\i test/psql.sql
\unset ECHO
1..3
ok 1 - Should create functions without errors
ok 2 - Function sch1.multiply(int, int) should exist
ok 3 - Function sch1.divide(int, int) should exist
